docs/src/Atmos/Microphysics.md

```diff
@@ -0,0 +1,369 @@
+\end{equation}
+```
+The default value of ``C_{drag}`` is chosen such that the ``v_t`` is close to the empirical terminal velocity formulation in Smolarkiewicz and Grabowski 1996. Assuming a constant drag coefficient is an approximation as it should be size and flow dependent, see [drag_coefficient](https://www.grc.nasa.gov/www/K-12/airplane/dragsphere.html).
+
+
+```@example rain_terminal_velocity
+using CLIMA.Microphysics
+using Plots
+
+# eq. 5d in Smolarkiewicz and Grabowski 1996
+# https://doi.org/10.1175/1520-0493(1996)124<0487:TTLSLM>2.0.CO;2
+function terminal_velocity_empirical(q_rai::DT, q_tot::DT, ρ::DT, ρ_air_ground::DT) where {DT<:Real}
+    rr  = q_rai / (DT(1) - q_tot)
+    vel = DT(14.34) * ρ_air_ground^DT(0.5) * ρ^-DT(0.3654) * rr^DT(0.1346)
+    return vel
+end
+
+q_rain_range = range(1e-8, stop=5e-3, length=100)
+ρ_air, q_tot, ρ_air_ground = 1.2, 20 * 1e-3, 1.22
+
+plot(q_rain_range * 1e3,  [terminal_velocity(q_rai, ρ_air) for q_rai in q_rain_range], xlabel="q_rain [g/kg]", ylabel="velocity [m/s]", title="Average terminal velocity of rain", label="CLIMA")
+plot!(q_rain_range * 1e3, [terminal_velocity_empirical(q_rai, q_tot, ρ_air, ρ_air_ground) for q_rai in q_rain_range], label="Empirical")
+savefig("rain_terminal_velocity.svg") # hide
+nothing # hide
+```
+![](rain_terminal_velocity.svg)
+
+## Cloud condensation/evaporation
+
+Condensation and evaporation of cloud water is parameterized as a relaxation to equilibrium. The equilibrium state is calculated using saturation adjustment defined in the `MoistThermodynamics` module.
+```math
+\begin{equation}
+  \left. \frac{d \; q_{liq}}{dt} \right|_{cond, evap} = \frac{q^{eq}_{liq} - q_{liq}}{\tau_{cond\_evap}}
+\end{equation}
+```
+where:
+ - ``q^{eq}_{liq}`` - liquid water specific humidity in equilibrium,
+ - ``q_{liq}`` - liquid water specific humidity,
+ - ``\tau_{cond\_evap}`` - relaxation timescale (parameter in `MicrophysicsParameters` module).
+
+## Autoconversion
+
+Autoconversion defines the rate of conversion form cloud to rain water due to collisions between cloud droplets. It is parameterized following Kessler 1995:
+```math
+\begin{equation}
+  \left. \frac{d \; q_{rai}}{dt} \right|_{acnv} = \frac{max(0, q_{liq} - q_{liq\_threshold})}{\tau_{acnv}}
+\end{equation}
+```
+where:
+ - ``q_{liq}`` - liquid water specific humidity,
+ - ``\tau_{acnv}`` - timescale (parameter in `MicrophysicsParameters` module),
+ - ``q_{liq\_threshold}`` - autoconversion (parameter in `MicrophysicsParameters` module).
+
+The default values of ``\tau_{acnv}`` and ``q_{liq\_threshold}`` are based on Smolarkiewicz and Grabowski 1996.
+
+## Accretion
+
+Accretion defines the rate of conversion from cloud to rain water resulting from collisions between cloud droplets and rain drops. It is parameterized following Kessler 1995:
+```math
+\begin{equation}
+\left. \frac{d \; q_{rai}}{dt} \right|_{accr} = \int_0^\infty n(r) \; \pi r^2 \; v_c(\rho) \; g^{1/2} \; r^{1/2} E_{col} q_{liq} dr
```

Comment:
Why not writing v_{drop} here in place of v_c(\rho) \; g^{1/2} \; r^{1/2} ? Seems clearer to me.

Reply:
done